.github/workflows/nightly.yml

```diff
@@ -0,0 +1,107 @@
+name: Nightly Build
+
+on:
+  schedule:
+  # Every night at 01:00
+  - cron: '0 1 * * *'
+  workflow_dispatch:
+    inputs:
+      release_branch:
+        description: Branch to release
+        required: true
+        default: master
+        type: string
+      quay_repository:
+        description: Quay repository
+        type: string
+        default: quay.io/kiali/kiali
+        required: true
+
+jobs:
+  initialize:
+    name: Initialize
+    runs-on: ubuntu-20.04
+    outputs:
+      target_branch: ${{ github.ref_name }}
+      quay_tag: ${{ steps.quay_tag.outputs.quay_tag }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.inputs.release_branch || github.ref_name }}
+
+    - name: Determine Quay tag
+      id: quay_tag
+      run: |
+        if [ -z ${{ github.event.inputs.quay_repository }} ];
+        then
+          QUAY_REPO="quay.io/leandroberetta/kiali"
```

Comment:
do you want your own here? Or do you mean for this to be `kiali/kiali` ?

Reply:
haha, no, I thought I changed this, SORRY. Now it's ok.